### ISSUE TITLE
Weird Changeling seeing through walls bug 

### ISSUE THREAD

**Issue by Bawhoppen** [tgstation/-tg-station#11773]
I was playing a round on basil as a changeling, anyways after I had already absorbed someone I was buying my powers and I misclicked, so I readapted and then I was somehow able to see through the walls like genetic's xray vision with darkness and all. 
![ss 2015-09-13 at 07 42 17](https://cloud.githubusercontent.com/assets/6510293/9840185/9eb7ae52-5a53-11e5-8b67-362b9e1d820e.png)
For the sake of science I tried killing myself and regenerating and that didn't remove it, and then it continued for the rest of the round.

**Comment by Bawhoppen** [comment]
I'll also add that I don't think it has to do with augmented eyesight, because I hadn't used it before the bug happened, and I tried to enable and disable afterward to no avail.

**Comment by PKPenguin321** [comment]
did you absorb and transform into somebody who had xray? iirc you can steal genetic powers of other people by transforming into them

**Comment by Bawhoppen** [comment]
I didn't transform into him, and I'm fairly sure he didn't have any powers as no real genetics happened, I'll check next time I see him on the server but 99% sure he didn't

**Comment by xxalpha** [comment]
Someone dun goofed with the flags again.

**Comment by CosmicScientist** [comment]
Do you recall any flavour messages such as "The walls disappear"?

**Comment by phil235** [comment]
Can't reproduce. I tried absorbing one person, then buying one random power and readapting right after.(and I had chemistry goggles on the whole time, just to be sure)

The only way to get xray is to absorb someone who has it and then transform into them (and you don't get any mutation activation message at all)

**Comment by xxalpha** [comment]
I tried reproducing and I couldn't do it either.

I did find that readapting doesn't clear the augmented eyesight if it is activated.

**Comment by Bawhoppen** [comment]
I was able to reproduce it on a test server and I'll tell you the exact order I did things in.
I spawned in, admin linged myself and then bought these powers (the ones I did originally and this time)
Aug eyesight, Fleshmend, Mimic voice, Armblade, Anatomic Panacea, Strained muscles, False Armblade sting.
Then I went to genetics and humanized a monkey (just incase spawned ones would be funky), Absorbed it, then readapted.
![ss 2015-09-14 at 09 22 28](https://cloud.githubusercontent.com/assets/6510293/9852130/29e29172-5ac2-11e5-8ecd-19bd5c9cee5b.png)

If you're still unable to do it I'll record a video of it.
